ci-operator/step-registry/openshift/microshift/infra/iso-boot/openshift-microshift-infra-iso-boot-commands.sh

```diff
@@ -134,9 +134,8 @@ done
 # already been copied into place in the iso-build step.
 SETTINGS_FILE="${SHARED_DIR}/scenario_settings.sh"
 cat <<EOF >"${SETTINGS_FILE}"
-PUBLIC_IP=${IP_ADDRESS}
-SSH_PUBLIC_KEY=~/.ssh/id_rsa.pub
-SSH_PRIVATE_KEY=~/.ssh/id_rsa
+SSH_PUBLIC_KEY=\${HOME}/.ssh/id_rsa.pub
```

Comment:
Is this because RF won't run within the container, but rather on the metal host directly?